src/abstractdatatable/reshape.jl

```diff
@@ -98,8 +98,8 @@ function stack(dt::AbstractDataTable, measure_vars::Vector{Int}, id_var::Int;
 end
 function stack(dt::AbstractDataTable, measure_var::Int, id_vars::Vector{Int};
                variable_name::Symbol=:variable, value_name::Symbol=:value)
-    stackdt(dt, [measure_var], id_vars;
-            variable_name=variable_name, value_name=value_name)
+    stack(dt, [measure_var], id_vars;
```

Comment:
Local tests say no, so it must be something I hadn't yet reverted. I'm not sure why this is the only stack function calling `stackdt`, but I'll investigate that when I update the indexing of stackdt's custom array views.